File: app/src/main/java/com/example/android_demo_application/utils/SharedPreferenceUtils.java
package com.example.android_demo_application.utils;

import android.content.SharedPreferences;

import com.example.android_demo_application.MyApplication;

public class SharedPreferenceUtils {
    private static String USER_NAME = "username";
    private static String PASSWORD = "password";

    public static SharedPreferences sharedPreferences  = MyApplication.getContext().getSharedPreferences("user_info",0);

    public static SharedPreferences.Editor editor = sharedPreferences.edit();
    public static String getSavedUserName(){
        return sharedPreferences.getString(USER_NAME,"");
    }
    public static String getSavedPassword(){
        return sharedPreferences.getString(PASSWORD,"");
    }
    public static void saveUserInfo(String username,String password){
        editor.putString(USER_NAME,username);
        editor.putString(PASSWORD,password);
        editor.apply();
    }

    public static void empty(){
        editor.clear();
        editor.apply();
    }
}

File: app/src/main/java/com/example/android_demo_application/utils/SharedPreferenceUtils.kt
package com.example.android_demo_application.utils

import com.example.android_demo_application.MyApplication

object SharedPreferenceUtils {

    private const val USER_NAME = "username"
    private const val PASSWORD = "password"
    var sharedPreferences = MyApplication.context?.getSharedPreferences("user_info", 0)
    var editor = sharedPreferences?.edit()
     val savedUserName: String?
        get() = sharedPreferences?.getString(USER_NAME, "")

   val savedPassword: String?
        get() = sharedPreferences?.getString(PASSWORD, "")

    @JvmStatic
    fun saveUserInfo(username: String?, password: String?) {
        editor!!.putString(USER_NAME, username)
        editor!!.putString(PASSWORD, password)
        editor!!.apply()
    }

    fun empty() {
        editor!!.clear()
        editor!!.apply()
    }
}